src/index.js

```diff
@@ -5,7 +5,7 @@ let bootstrapQueue = [];
 let sharedLoader = null;
 let Aurelia = null;
 const host = PLATFORM.global;
-const isNodeLike = typeof process !== 'undefined' && !process.browser;
+const isNodeLike = typeof process !== 'undefined' && !process.browser && !process.versions['nw'];
```

Comment:
I think it would be better to do the change here instead:
https://github.com/timfish/bootstrapper/blob/6cb43ef2c87c0e776acb4360fbc540b9f4936650/src/index.js#L87

Rename `isElectronRenderer` to something like `isRenderer` and set it to true if we're either inside an Electron Renderer or an NwJS instance.